content/blog/multi-stage-yaml-pipeline-to-create-and-publish-nuget-packages.md

```diff
@@ -0,0 +1,263 @@
+---
+title: "Multi-stage YAML pipeline to create and publish NuGet packages"
```

Comment:
Heel klein, maar ik zou using in de titel zetten. Dus: Using multi-stage yaml pipelines to create and publish nuget packages